<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IdleNotificationWidget</class>
 <widget class="QWidget" name="IdleNotificationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>254</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(85, 85, 85);font-size:13px;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="idleSince">
     <property name="text">
      <string>You have been idle since 12:34:56</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="idleDuration">
     <property name="text">
      <string>(5 minutes)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="runningTimeEntryLabel">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Running time entry:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timeEntryDescriptionLabel">
     <property name="text">
      <string>Doing the thing</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="keepTimeButton">
     <property name="text">
      <string>Keep time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="discardTimeButton">
     <property name="text">
      <string>Discard time</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="discardTimeAndContinueButton">
     <property name="text">
      <string>Discard time and continue</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add idle time as a new time entry</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
